js/pihole/index.js

```diff
@@ -110,11 +134,39 @@ function updateSummaryData(runOnce) {
 
 function updateQueriesOverTime() {
     $.getJSON("api.php?overTimeData", function(data) {
+        // Add data for each hour that is available
         for (hour in data.ads_over_time) {
-            timeLineChart.addData([data.domains_over_time[hour], data.ads_over_time[hour]], hour + ":00");
+            // Add x-axis label
```

Comment:
The *.addData routine is not available anymore. I now do it like this to achieve the same behavior.